62_CAD/common.hlsl

```diff
@@ -41,14 +42,14 @@ struct DrawObject
 
 struct LinePointInfo
 {
-    float64_t2 p;
+    nbl::hlsl::portable_vector64_t2 p;
```

Comment:
a bit ugly but I think this is necessary cause other choice would be for this to stay a float64_t2 and then in vtx shader construct a portable_vector64_t2 when loading these values and I assume constructing one is expensive (when float64 feature is disabled)

Reply:
you cannot have any `float64_t` (except CONSTANT LITERALS or anything else that gets constant folded into not-a-float64) if you want to avoid emitting the `shaderFloat64` capability in the SPIR-V

Reply:
mostly a reply to @Erfan-Ahmadi , but thanks for the emote @Przemog1